server/game/cards/06-WoE/NirborFlamewing.js

```diff
@@ -0,0 +1,76 @@
+const Card = require('../../Card.js');
+
+class NirborFlamewing extends Card {
+    // At the start of your turn, if Nirbor Flamewing is in your
+    // discard pile, you may destroy a friendly creature. If you do,
+    // put Nirbor Flamewing into play ready.
+    //
+    // Destroyed: Make a token creature.
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onBeginRound: (_, context) => context.player === this.game.activePlayer
+            },
+            optional: true,
+            location: 'discard',
+            effectStyle: 'all',
+            target: {
+                activePromptTitle: 'Choose a creature',
+                cardType: 'creature',
+                controller: 'self',
+                gameAction: ability.actions.destroy()
+            },
+            then: {
+                message: '{0} uses {1} to put {1} into play ready',
+                gameAction: ability.actions.putIntoPlay({
+                    target: this,
+                    ready: true
+                })
+            }
+        });
+
+        // If Nirbor dies during the start of turn pahase (e.g., to
+        // another Nurbor, or by General Order 24, etc), we should have the
+        // option of immediately bringing it back.  But we can't fire
+        // immediately on the destroyed event in the case of 2 Nirbors
+        // because then we won't get to the `then` clause. The
+        // `addEventToWindow` call is trying to get around that but it's
+        // not quite right.
+        this.reaction({
```

Comment:
This case is broken for the 2x Nirbor scenario but it might be good enough to go with for now, and we can revisit later.  Or, if you know how to fix it, please let me know!